.changeset/red-cameras-burn.md

```diff
@@ -0,0 +1,6 @@
+---
+'@craftjs/layers': minor
+'@craftjs/core': patch
+---
```

Comment:
1. I don't think the core package needs an update since the changes here only affect the layers package?
2. Can you change the `@craftjs/layers` as a `patch` update instead? It's not ideal but the reason being we're still in major version 0 and we use the minor version to indicate a breaking change in Craft